src/app/components/grid/index.html

```diff
@@ -321,6 +324,7 @@
 			<ul>
 				<li>The list is presented in a tile, page section, or other container that might flow off the screen.</li>
 			</ul>
+			<p>Include the multiselect toolbar to allow users to select or clear all items and optionally view only the selected items.</p>
       <p>When users select rows, the summary action bar should fly in from the bottom of the page. Place actions that can be taken on the selected items in the summary action bar.</p>
```

Comment:
I know this isn't new content, but I have the same question here as in the comment on the summary action bar change. Should "should fly in" here be "flies in"?

Reply:
Resolving based on response to the other comment.